src/orders/components/OrderInvoiceList/OrderInvoiceList.tsx

```diff
@@ -37,9 +37,9 @@ const useStyles = makeStyles(
 
 interface OrderInvoiceListProps {
   invoices: InvoiceFragment[];
-  onInvoiceGenerate?: () => void;
-  onInvoiceClick?: (invoice: InvoiceFragment) => void;
-  onInvoiceSend?: (invoice: InvoiceFragment) => void;
+  onInvoiceGenerate: () => void;
+  onInvoiceClick: (invoice: InvoiceFragment) => void;
+  onInvoiceSend: (invoice: InvoiceFragment) => void;
```

Comment:
Everywhere else we use ids so let's keep it that way

```suggestion
  onInvoiceClick: (invoiceId: string) => void;
  onInvoiceSend: (invoiceId: string) => void;
```